lib/src/analysis_options.dart

```diff
@@ -48,3 +52,35 @@ linter:
     - unrelated_type_equality_checks
     - valid_regexps
 ''';
+
+const _analyzerErrorKeys = const <String>['uri_has_not_been_generated'];
+
+String customizeAnalysisOptions(String original, bool isFlutter) {
```

Comment:
TODO: add an unit test for this – since it's more than a bit complicated

Reply:
@isoos – did you sanity check it w/ one of the samples we care about?

Reply:
Added tests, which caught an edge-case that was not caught with the sampling manual test.

Reply:
😄